packages/redux/src/entities/reducer/entitiesMapper.ts

```diff
@@ -8,8 +8,8 @@ import { entitiesMapper as merchantsLocationsEntitiesMapper } from '../../mercha
 import { entitiesMapper as ordersEntitiesMapper } from '@farfetch/blackout-client/orders/redux';
 import { entitiesMapper as paymentsEntitiesMapper } from '../../payments';
 import { entitiesMapper as productsEntitiesMapper } from '../../products';
-// @ts-expect-error ts-migrate(2307) FIXME: Cannot find module '@farfetch/blackout-client/profile/redux' or it... Remove this comment to see the full error message
-import { entitiesMapper as profileEntitiesMapper } from '@farfetch/blackout-client/profile/redux';
+// @ts-expect-error ts-migrate(2307) FIXME: Cannot find module '@farfetch/blackout-redux/users' or it... Remove this comment to see the full error message
+import { entitiesMapper as usersEntitiesMapper } from '@farfetch/blackout-redux/users';
```

Comment:
Following the other imports shouldn't this be `(...) from '../../users'`?

Reply:
Changed it 👍